### DISCUSSION TITLE
Foci for 2.4.x?

### DISCUSSION THREAD

**Discussion by dankamongmen** [dankamongmen/notcurses#2186] [General]
What ought I be working on? Where are there pain points?

Things that seem important in the short-to-medium term include:
* Complete support for kitty keyboard protocol
* Do some testing and bugfixing with Wezterm, where I think we have some issues
* Finish out Linux framebuffer graphics, where we have some bugs
* Work out kinks from `ncneofetch`, where we have some bugs, especially on macOS
* Move `ncls` to CLI mode, and make it make better use of horizontal space
* Further work on Microsoft Windows, where I'm sure we have some issues (researchy)
* Improve Sixel quantization (complex)
* Smooth bitmap movement (complex)

In the longer term, mosaics (very complex) seem the most important thing IMHO. I think they're probably post-3.0.0 material. So I guess this is a list of things wanting done before 3.0.0 drops?

**Comment by masflam** [comment]
I'm all for Linux console graphics and the kitty keyboard proto myself. Ooh I wanna see my 3D renderer in action on the tty.

**Reply by masflam** [reply]
And I shall actually profile it to see whether my code, notcurses code, or the terminal's throughput is the main performance hit.
